src/includes/group.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * An event whose payload type extends this interface may be associated
+ * in some way with a particular search box.
```

Comment:
Is this still true? It sounds like we've abstracted `group` away from the idea of linking to a search box.

Reply:
Good catch. I missed this one.